op-chain-ops/cmd/check-l2/main.go

```diff
@@ -684,6 +687,24 @@ func checkOptimismMintableERC20Factory(addr common.Address, client *ethclient.Cl
 	}
 	log.Info("OptimismMintableERC20Factory", "bridge", bridge.Hex())
 
+	initialized, err := getInitialized("OptimismMintableERC20Factory", addr, client)
+	if err != nil {
```

Comment:
Could you check if `checkAlreadyInitialized` does this assertion? We could likely replace the assertion `if initialized.Uint64() != 1 {`